src/routes/email/email.controller.delete-email.spec.ts

```diff
@@ -242,20 +238,20 @@ describe('Email controller delete email tests', () => {
           return Promise.reject(new Error(`Could not match ${url}`));
       }
     });
-    emailDatasource.getEmail.mockResolvedValueOnce(email);
+    accountDataSource.getAccount.mockResolvedValueOnce(account);
     emailApi.deleteEmailAddress.mockRejectedValue(new Error('Some error'));
 
     await request(app.getHttpServer())
       .delete(`/v1/chains/${chain.chainId}/safes/${safe.address}/emails`)
       .send({
-        account: account.address,
+        account: signer.address,
```

Comment:
On the email routes, `account` is used. I'm impartial as to whether we keep them as `account` but wanted to point out that they are no longer the same as what is mentioned in errors (`signer`).

Reply:
I agree. Will update the API spec accordingly.

Changed in [d56dd44](https://github.com/safe-global/safe-client-gateway/pull/1050/commits/d56dd4443b128abb5be9c79161c5d9b8bcc4b8ed)